mne/viz/ica.py

```diff
@@ -787,6 +787,8 @@ def _plot_sources_raw(ica, raw, picks, exclude, start, stop, show, title,
     inds = list(range(len(picks)))
     data = np.array(data)
     n_channels = min([20, len(picks)])
+    raw._first_time = raw.first_samp / float(raw.info['sfreq'])
```

Comment:
we shouldn't monkey-patch an object just for this. It's not very expensive to recompute it below